client/app/styles/modules/_m-actions-milestones.scss

```diff
@@ -56,41 +52,6 @@
   }
 }
 
-.milestone.present + .milestone.future {
```

Comment:
Removing for now.
1) It wasn't stable because the assigning of `.present` and `.future` classes to milestone elements wasn't stable. (due to `tense()` cp instability)

2) we shouldn't rely on CSS to calculate the position of a "now" marker.

 In the future we can work out some component to insert a "now" marker.